packages/ui/src/components/QueryBuilder/Header/Tools/SavedFiltersMenu.tsx

```diff
@@ -32,7 +32,7 @@ const SavedFiltersMenu = ({
     const { dictionary } = useContext(QueryBuilderContext);
     const [manageFiltersVisible, setManageFiltersVisible] = useState(false);
 
-    const confirmUnsavedChangeForExistingFilter = (onOkCallback: Function) => {
+    const confirmUnsavedChangeForExistingFilter = (onOkCallback: any) => {
```

Comment:
On pourrait typer void non ?
```suggestion
    const confirmUnsavedChangeForExistingFilter = (onOkCallback: () => void) => {
```